debian/control

```diff
@@ -117,3 +118,17 @@ Description: WebDriver driver for the Chromium Browser
  WebDriver.
  .
  See http://code.google.com/p/selenium/wiki/ChromeDriver for details.
+
+Package: chromium-browser-appmode
+Architecture: any
+Pre-Depends: dpkg (>= 1.15.6)
+Depends: ${shlibs:Depends},
+	${misc:Depends},
+	chromium-browser,
+	python3,
+	python3-gi
+Provides: mime<x-scheme-handler/webapp>
+Description: Installs a script to handle webapp://<WM_CLASS>@<URI> URIs
+ This allows us to easily run chromium in application mode from inside
```

Comment:
I don't think I'd mention gvfs-open because what it's needed for gvfs-open to work is the OpenURI portal, and this is simply a handler that provides a way to deal with webapp:// schemens, either from inside or outside the sandbox.

That said, I agree the comment is not generic enough, as it transmits the idea that it's only useful for sandboxed applications, which is not correct (as it's also useful for web links, which do not use flatpak at all).

I'll re-word it